Sample-XML/Position Limits.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Snapshot Date="2017-01-03">
+  <Instruments>
+
+	<Future InstrumentID="11" InstrumentName="AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" Market="CME" DeliveryMonth="2017-03" CommoditySymbol="POG" IsCashSettled="true"/>

```

Comment:
super - thought so :) Yeah we can get rid of @aliyaladhani no need to add additional stuff we don't need atm